tests/utils/exec/occli/cmd_client.go

```diff
@@ -64,7 +69,27 @@ func RunCMD(cmd string) (stdout string, stderr string, err error) {
 	return
 }
 
-func DummyGetKubeConfigFromEnv() {
+func DummyGetKubeConfigFromEnv() error {
 	// login with oc cmd and then get the kubeconfig from  ~/.kube/config
 	// Refer to OCM-9183
+	usersValue, err := helper.ReadFileContent(config.Test.ClusterKubeconfigIdpFile)
```

Comment:
It would be better to set it in function parameter other than use the env values directly